mayastor/src/bdev/dev.rs

```diff
@@ -47,6 +48,7 @@ impl Uri {
         match url.scheme() {
             // backend NVMF target - fairly unstable (as of Linux 5.2)
             "nvmf" => Ok(Box::new(nvmx::NvmfDeviceTemplate::try_from(&url)?)),
+            "malloc" => Ok(Box::new(malloc::Malloc::try_from(&url)?)),
```

Comment:
This file looks quite different from the upstream version. If it's going to replace it, the comment for the malloc URI should be copied across.

Reply:
We will restore all other required URI schemes when merging with the upstream. Now its a WIP.